examples/flow-counter/.flowconfig

```diff
@@ -0,0 +1,16 @@
+[ignore]
+.*/node_modules/
+# .*/node_modules/webpack/.*
+# .*/node_modules/redux/examples/.*
+# .*/node_modules/redux/lib/.*
+# Ignoring analyzing redux for now since I can't find a way for it to not also select redux-thunk (need a better regex)
+# module.name_mapper='^\(redux\)$' -> '\1/src/index'
+
+[include]
+
+[libs]
+interfaces/
+
+
+[options]
+module.system=node
```

Comment:
As you can see, I removed the line `module.name_mapper='^\(redux\)$' -> '\1/src/index'` and opted to continue with Flow declaration modules (the `[libs]` option) since that regex was conflicting with `redux-thunk`, and I was not able to create a regex that wouldn't stop that problem. If anyone can give this a try, or discuss approaches, let's start a thread here.